src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -1691,7 +1691,15 @@ void program_node::create_onednn_primitive_attributes(
                         } else {
                             auto in_scale = get_input_layout(dep_idx++);
                             resize_layout_for_fc(this, in_scale);
-                            dnnl::memory::desc in_scale_desc = onednn::layout_to_memory_desc(in_scale, onednn::get_default_data_format(in_scale));
+
+                            dnnl::memory::desc in_scale_desc;
+                            if (is_type<gemm>() || is_type<fully_connected>()) {
+                                in_scale_desc = onednn::layout_to_memory_desc(in_scale, onednn::get_default_data_format(in_scale));
```

Comment:
Don't gemm and fc need need_blocked when blocked?

Reply:
Sorry but it is difficult to understand the logic. Let's discuss offline together with @jade-cho .

* @jade-cho  is "need_blocked" proper name? Maybe "allow_blocked" is the right name? actually the behavior seems similar to `!flatten` :(
* why should we treat `gemm and fc` differently? If test-case fails, we may change the test case.
* Maybe should we implement the logic withint the function itself?